### DISCUSSION TITLE
Fetch example schema after putting #[schema(example = "...")] for all its elements

### DISCUSSION THREAD

**Discussion by TheAwiteb** [juhaku/utoipa#484] [Q&A] [answered]
Hi @juhaku thank you for your hard work, I really admire the work you've done.
My question is how can I extract the example for the schema. Because I want to put it in the response example
```rust
responses(
    (
        status = 200, description = "...", body = FooSchema,
        example = json!{FooSchema::example()}
        //         |   ^^^^^^^^^^^^^^^^^^^^^^
        //         |____^ Here
    )
),
```

**Comment by juhaku** [comment, marked as answer]
Hi, 
And thanks, :slightly_smiling_face: 

Simply put, there is no way to do that, unfortunately. Currently the only way to provide example is to call `json!` with new object or  with json syntax itself the same way one would call `serde_json::json!(...)`. 

To implement this support will need some thinking but perhaps could be added in future.
